core/src/components/datetime/datetime.md.scss

```diff
@@ -30,15 +30,43 @@
 
 // Calendar / Header / Action Buttons
 // -----------------------------------
-:host .datetime-calendar .calendar-action-buttons ion-item {
-  --padding-start: #{$datetime-md-header-padding};
+
+:host .calendar-action-buttons .calendar-month-year-toggle {
+  color: #{$text-color-step-350};
```

Comment:
Can this be folded into the `.calendar-month-year-toggle` rule below?

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/28443/commits/ccd7a4223a6a4b1a2c20c2c6ef5a19982a2dde03